### ISSUE TITLE
Update setup instructions for neovim

### ISSUE THREAD

**Issue by ewen-lbh** [hyprland-community/hyprls#12]
- [ ] Since we have proper lspconfig, get rid of the ugly autocmd thing (also see #9)
- [ ] Mention astronvim support (see https://github.com/AstroNvim/astrocommunity/pull/1063)
- [ ] Mention Mason LSPConfig support (see https://github.com/williamboman/mason-lspconfig.nvim/pull/428)

**Comment by igorlfs** [comment]
By the way, it would be interesting to bundle binaries in the releases page, so the integration with Mason could be streamlined. I wanna try hyprls, but since I don't use Go myself, the process is slightly more annoying, since I have to use my system's package manager to install Go and then build from source.

**Comment by ewen-lbh** [comment]
Yeah including built binaries with releases would be nice, I'll add a task for that

**Comment by arthsmn** [comment]
If you want any help with the Nix part, just ask. I can also help adding a flake.nix in the repository to help with development and for people to be able to test the latest commit.

**Comment by ewen-lbh** [comment]
thx for the suggestion! although unless this gets distributed on more package managers, a repology badge is kinda... weird haha

would totally merge a pr that adds a flake.nix though!
